larray/core/array.py

```diff
@@ -5081,23 +5081,27 @@ def __float__(self):
         return self.data.__float__()
 
     @deprecate_kwarg('nan_equals', 'nans_equal')
-    def equals(self, other, rtol=0, atol=0, nans_equal=False):
-        """
+    def equals(self, other, rtol=0, atol=0, nans_equal=False, check_axes=False):
+        r"""
         Compares self with another array and returns True if they have the same axes and elements, False otherwise.
 
         Parameters
         ----------
-        other: LArray-like
+        other : LArray-like
             Input array. aslarray() is used on a non-LArray input.
         rtol : float or int, optional
             The relative tolerance parameter (see Notes). Defaults to 0.
         atol : float or int, optional
             The absolute tolerance parameter (see Notes). Defaults to 0.
-        nans_equal: boolean, optional
+        nans_equal : boolean, optional
             Whether or not to consider nan values at the same positions in the two arrays as equal.
             By default, an array containing nan values is never equal to another array, even if that other array
             also contains nan values at the same positions. The reason is that a nan value is different from
             *anything*, including itself. Defaults to False.
+        check_axes : boolean, optional
+            Whether or not to check that the set of axes and their order is the same on both sides. Defaults to True.
```

Comment:
s/Defaults to True/Defaults to False

Reply:
nice catch ! That's what you get for inverting the default at the last minute :)

Reply:
or working without specifications